src/plugins/Plugin.js

```diff
@@ -59,6 +59,13 @@ module.exports = class Plugin {
     if (targetElement) {
       this.core.log(`Installing ${callerPluginName} to a DOM element`)
 
+      // attempt to extract meta from form element
+      if (this.opts.getMetaDataFromForm && targetElement.nodeName === 'FORM') {
+        const formMeta = getFormData(targetElement)
+        this.core.log('Adding metadata from form')
+        this.core.log(formMeta)
```

Comment:
🙀 I don’t know how this happened, I was checking `state.meta` after every change.